src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -84,6 +84,11 @@ export class Message extends React.Component<Props, any> {
   render() {
     const { artworkPreview, initials, message, senderName } = this.props
     const isSent = this.props.relay ? !this.props.relay.hasOptimisticUpdate(message) : true
+    const transactions = this.props.relay.getPendingTransactions(message)
+
+    if (transactions && transactions.length) {
```

Comment:
This is never anything but `COMMITTING`; I never get a `COMMIT_FAILED`. See https://facebook.github.io/relay/docs/api-reference-relay-container.html#getpendingtransactions-example